[Product Analytics] User Segmentation (#23568)
DataDog/documentation: content/en/product_analytics/segmentation/_index.md

---
title: Segmentation
kind: documentation
description: Analyze and understand specific groups or segments of your user base.
further_reading:
- link: "/product_analytics/"
  tag: "Documentation"
  text: "Product Analytics"
---

## Overview

> Segmentation is a feature in Product Analytics that allows you to analyze and understand specific groups or segments of your user base. By segmenting your users based on various characteristics and behaviors, you can uncover valuable insights, identify trends, and make data-driven decisions to optimize your product and user experience. For example, you can create segment users by purchase amount, active users in a given country, users in a trial, or users in a trial who then converted to a paying organization.

Reviewer: ```suggestion
Segmentation is a feature in Product Analytics that allows you to analyze and understand specific groups or segments of your user base. By segmenting your users based on various characteristics and behaviors, you can uncover valuable insights, identify trends, and make data-driven decisions to optimize your product and user experience. For example, you can segment users by purchase amount, active users in a given country, users in a trial, or users in a trial who then converted to a paying organization.
```

## Build a segment

To create a new segment, navigate to **[Digital Experience Monitoring > Product Analytics > User Segments][1]** from the navigation. You have two sources you can choose from when building a new segment:

> - RUM data

Author: ```suggestion
- Product Analytics data
```

> - RUM data

Reviewer 2: anytime it says RUM, can we update to Product Analytics @rtrieu ?

- External data (upload a Reference Table)

> {{< img src="product_analytics/segmentation/segmentation-1.png" alt="Create a user segment based on RUM or external data.">}}

Author: ```suggestion
{{< img src="product_analytics/segmentation/segmentation-1.png" alt="Create a user segment based on Product Analytics or external data.">}}
```

> ### Using RUM data

Author: ```suggestion
### Using Product Analytics data
```

> To create a new segment using RUM data:

Author: ```suggestion
To create a new segment using Product Analytics data:
```

> 1. Choose the user attribute you want to connect your data with. In the below example, it is `usr.id`, but you can use any user attribute available, such as (`usr.email` or `usr.name`).

Reviewer: Are the parentheses here placed correctly? I'd want to interpret this as users being able to connect on an ORed combination `(usr.email or usr.name)`, in which case I'd put the whole parenthetical in backticks. Otherwise, the parentheses should be removed.

> 1. Choose the user attribute you want to connect your data with. In the below example, it is `usr.id`, but you can use any user attribute available, such as (`usr.email` or `usr.name`).

Author: ```suggestion
1. Choose the user attribute you want to connect your data with. In the below example, it is `usr.id`, but you can use any user attribute available, such as `usr.email` or `usr.name`.
```

> 2. Under the **Filter your segment** section, you can filter on any attribute collected by the SDK, or custom ones you added to create a granular user segment.

Reviewer: ```suggestion
2. Under the **Filter your segment** section, you can filter on any attribute collected by the SDK, or custom attributes you added to create a granular user segment.
```

>    In the below example, we filtered to all users who were on the `/cart` page, then clicked on the checkout button, and did so from Brazil:

Reviewer: ```suggestion
   In the below example, the segment is filtered to all users who were on the `/cart` page and then clicked on the checkout button (and did so from Brazil):
```

   {{< img src="product_analytics/segmentation/segmentation-2.png" alt="Filter to all users from Brazil who were on the `/cart` page and clicked on the checkout button.">}}

### Using external or third-party data

To create a segment using external or third-party data:

> 1. See the [Reference Tables][2] documentation for how to upload and create a reference table.

Reviewer: ```suggestion
1. See the [Reference Tables][2] documentation for how to upload and create a Reference Table.
```
seems we capitalize this

2. Make sure to properly connect the data type (for example, `usr.id`, `usr.name`, or `usr.email`) to the column name.

> 3. Check to ensure there is data in Product Analytics for users in the segment for accuracy.

Reviewer: ```suggestion
3. For accuracy, ensure there is data in Product Analytics for users in the segment.
```

> ## Use segments in the Analytics Explorer

Reviewer: is this what this heading is supposed to say? I'd expect something more generic like "Visualize segments in Datadog". Also, the next two headings should probably be h3.

Reviewer 2: +1 maybe something like "Ways to leverage Segments across Product Analytics"

Author: Updated to "Leverage segments across Product Analytics"

> ## Use segments in the Analytics Explorer

Author: ```suggestion
## Ways to leverage Segments across Product Analytics
```

> ## Sankey

Author: ```suggestion
### Sankey
```

> ## Ways to leverage Segments across Product Analytics

Author: ```suggestion
## Leverage segments across Product Analytics
```

> ### In sankeys

Reviewer 3: I noticed there's a beta label on this page: https://docs.datadoghq.com/real_user_monitoring/product_analytics/sankey/
Is it GA for Product Analytics this feature?

Author: That will be removed in the larger PR for PA :)

> ### Sankey

Author: ```suggestion
### In sankeys
```

> In the Sankey page, you can filter down the visualization to reflect a selected segment of users. This allows you to look at the experience and traffic patterns of a particular set of users in a given segment. The below example shows a Sankey only for users in the "Premium users" segment.

Reviewer: ```suggestion
In the Sankey page, you can filter the data in the visualization to reflect a selected segment of users. This allows you to look at the experience and traffic patterns of a particular set of users in a given segment. The below example shows a Sankey diagram only for users in the "Premium users" segment.
```

{{< img src="product_analytics/segmentation/segmentation-3.png" alt="Filter down the Sankey visualization to reflect a selected segment of users.">}}

> ### Analytics Explorer

Author: ```suggestion
### In Analytics Explorer
```

> You can filter down the visualization in the Analytics Explorer to reflect a selected set of users that are in a given segment. The example below shows a list of users in the "Premium users" segment who were active in the last month, organized by the number of sessions they had.

Reviewer: ```suggestion
You can filter the data in the Analytics Explorer to reflect a selected set of users that are in a given segment. The example below shows a list of users in the "Premium users" segment who were active in the last month, organized by the number of sessions they had.
```
